src/components/ProductListing/ProductListing.js

```diff
@@ -1,5 +1,6 @@
 import { useNavigate } from "react-router-dom";
-import { useCart } from "../../contexts";
+import { useCart, useWishlist } from "../../contexts";
+import { removeFromWishlistHandler } from "../../utilities/wishlist/removeFromWishlist";
```

Comment:
if possible use absolute path to import files

Reply:
okk will do in refactoring path